src/ice/agent.rs

```diff
@@ -382,6 +382,27 @@ impl IceAgent {
         }
     }
 
+    /// Determine whether an equivalent remote candidate is part of a viable candidate pair.
+    pub fn find_viable_pair_for_equivalent_remote_candidate(
+        &self,
+        c: &Candidate,
+    ) -> Option<&CandidatePair> {
+        self.candidate_pairs
+            .iter()
+            .filter(|cand| cand.state() == CheckState::Succeeded)
+            .find_map(|pair| {
+                let o = &self.remote_candidates[pair.remote_idx()];
+
+                let same = c.addr() == o.addr()
```

Comment:
Maybe this business should be encapsulated in some method `Candidate::equivalent`

Reply:
Maybe. But let's revise that in another PR.